app/namespace.go

```diff
@@ -57,6 +60,38 @@ var (
 		"us-east-1",
 		"us-west-2",
 	}
+	sinkNameFlag = &cli.StringFlag{
+		Name:     "sink-name",
+		Usage:    "sink name",
+		Required: true,
+	}
+	sinkEnabledFlag = &cli.BoolFlag{
+		Name:  "enabled",
+		Usage: "Whether the export sink is enabled",
+	}
+	sinkTypeFlag = &cli.StringFlag{
```

Comment:
is this meant to be destination type? we might want to reword or indicate that. Also I assume we will only support limited set of values (in this case, only S3?)